src/main/java/com/zamoiski/dao/DepartmentMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.zamoiski.dao;
+
+import com.zamoiski.entity.Department;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DepartmentMapper implements RowMapper<Department> {
+    @Override
+    public Department mapRow(ResultSet rs, int i) throws SQLException {
+        Department department = new Department();
+        department.setId(rs.getLong("id"));
+        department.setDepartment_name(rs.getString("department_name"));
+        department.setDateOfCreate(rs.getTimestamp("date_of_create").toLocalDateTime());
+        return department;
+    }
+}
```

Comment:
remove class, just use lambda as method in dao like  `private RowMapper<Department> departmentRowMapper =(resultSet, i)->new Department(....`